yarn is running postinstall from npm package

Crap, https://github.com/fable-compiler/ts2fable/pull/129 changes messed up installs Guess I was wrong here: https://github.com/fable-compiler/ts2fable/pull/129#discussion_r153169188

```
C:\Users\camer> yarn global add ts2fable@next
yarn global v1.3.2
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.1.3: The platform "win32" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
warning ts2fable@0.5.0-build.142: The engine "vscode" appears to be invalid.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
[1/1] ⡀ ts2fable
[-/1] ⡀ waiting...
[-/1] ⡀ waiting...
[-/1] ⡀ waiting...
error C:\Users\camer\AppData\Local\Yarn\config\global\node_modules\ts2fable: Command failed.
Exit code: 1
Command: node ./node_modules/vscode/bin/install
Arguments:
Directory: C:\Users\camer\AppData\Local\Yarn\config\global\node_modules\ts2fable
Output:
module.js:544
    throw err;
    ^

Error: Cannot find module 'C:\Users\camer\AppData\Local\Yarn\config\global\node_modules\ts2fable\node_modules\vscode\bin
    at Function.Module._resolveFilename (module.js:542:15)
    at Function.Module._load (module.js:472:25)
```

--- a/package.json
+++ b/package.json
@@ -12,8 +12,7 @@
   "scripts": {
     "build": "fable-splitter -c splitter.config.js",
     "watch": "fable-splitter -c splitter.config.js -w",
-    "test": "fable-splitter -c splitter.test.config.js",
-    "postinstall": "node ./node_modules/vscode/bin/install"
+    "test": "fable-splitter -c splitter.test.config.js"
   },
   "repository": {
     "type": "git",
@@ -57,8 +56,5 @@
     "mocha": "^4.0.1",
     "mocha-appveyor-reporter": "^0.4.0",
     "vscode": "^1.1.8"
-  },
-  "engines": {
-    "vscode": "1.18.1"
   }
 }

--- a/src/ts2fable.fs
+++ b/src/ts2fable.fs
@@ -106,7 +106,7 @@ if argv |> List.exists (fun s -> s = "splitter.config.js") then // run from buil
     writeFile ["node_modules/@types/yargs/index.d.ts"] "test-compile/Yargs.fs"
 
     // for test-compile
-    writeFile ["node_modules/vscode/vscode.d.ts"] "test-compile/VSCode.fs"
+    // writeFile ["node_modules/vscode/vscode.d.ts"] "test-compile/VSCode.fs"
     writeFile ["node_modules/izitoast/dist/izitoast/izitoast.d.ts"] "test-compile/IziToast.fs"
     writeFile ["node_modules/electron/electron.d.ts"] "test-compile/Electron.fs"
     writeFile ["node_modules/@types/react/index.d.ts"] "test-compile/React.fs"

--- a/test-compile/test-compile.fsproj
+++ b/test-compile/test-compile.fsproj
@@ -7,7 +7,7 @@
     <Compile Include="Node.fs" />
     <Compile Include="Yargs.fs" />
 
-    <Compile Include="VSCode.fs" />
+    <!-- <Compile Include="VSCode.fs" /> -->
     <Compile Include="IziToast.fs" />
     <!-- <Compile Include="Electron.fs" /> -->
     <!-- <Compile Include="React.fs" /> -->

